Tanvir04Khan/01-max-udemy/03-login-page/src/Components/Home/HomePage.js

```diff
@@ -0,0 +1,24 @@
+import React, { Fragment } from "react";
+
+import HomeHeader from "./HomeHeader";
+import Card from "../UI/Card";
+
+import "./HomePage.css";
+
+const HomePage = ({ isLoggedin }) => {
+  const LogoutBtnHandler = () => {
+    isLoggedin(false);
+    localStorage.removeItem("isLoggedIn");
+  };
+  return (
+    <Fragment>
+      <HomeHeader onLoggedout={LogoutBtnHandler} />
+      <Card>
+        Welcome Back!
+        <button onClick={LogoutBtnHandler}>Logout</button>
```

Comment:
respect casing while creating variables, function , components etc

Reply:
> changed code as per suggestion @faran4engg

where ? I still see function name starting with capital Letter ??
also I think it should be `handleLogout`

Reply:
changed code as per suggestion @faran4engg